Apps/Src/main.c

```diff
@@ -16,11 +16,11 @@
 #include "BSP_Lights.h"
 #include "BSP_WDTimer.h"
 
-
 cell_asic Minions[NUM_MINIONS];
 bool override = false;		// This will be changed by user via CLI
 char command[COMMAND_SIZE];
 
+void heartbeat(void);
 void initialize(void);
 void preliminaryCheck(void);
 void faultCondition(void);
```

Comment:
We haven't really been good at this, but I realized that all these functions could probably benefit from being marked `static`.